ui/src/pages/ViewEvent.jsx

```diff
@@ -37,7 +63,7 @@ class ViewEvent extends React.Component {
             <Image src={event.images[0]} />
             <div className="attend-button">
               { /* TODO @helenhyewonlee: only show the attend button if the event is not in the trip */ }
-              <Button blue label="Attend" onClick={() => console.log("Attend button clicked")} />
+              <Button blue label="Attend" onClick={this.showAddPage} />
```

Comment:
We should modify this component to only show the attend button if we pass in a `showAttendButton` prop. Then change `ViewDiscoverEvent` and `ViewTripEvent` to pass the prop as `true` and `false` respectively. This way we don't see the button for `ViewTripEvent`

Reply:
@helenhyewonlee We can use the same logic to decide whether or not we want to show the `Edit` button?

Reply:
Yup! I'm still working on it, but in `ViewTripEvent`, I'm calling `<ViewEvent event = {event} isTripEvent={true} />` and in `ViewDiscoverEvent`, I'm calling `<ViewEvent event = {this.props.event} isTripEvent={false} />`